### DISCUSSION TITLE
Recording multiple clips (sub + main) from single camera

### DISCUSSION THREAD

**Discussion by kwanm84** [blakeblackshear/frigate#1701] [Ask A Question]
I am just starting to play with frigate and I'm loving it so far. I wanted to record clips from both the sub stream and main stream for my reolink cameras at the same time.

I tried setting it up in the config, and was told it couldn't be done by frigate. However that was in 0.8... I was wondering if its possible if anyone can test in 0.9. 

You may wonder why I want to do this.... the main stream is 4k h265, and while I can play those clips in edge it seems on my laptop (only just found this out which is awesome), I can't play them over home assistant app, or my mobile browser (haven't really done much research). If both clips could be recorded, I can watch the low res clips immediately on my phone, but still have the hi res ready to go and look at on the laptop.

If its not possible, could anyone indicate if this could be a feature request and if overheads would make it feasible recording the extra stream.

Thanks

Mike

**Comment by blakeblackshear** [comment]
It won't be possible to do this in 0.9.0. You could look at running something like moonfire nvr separately for the high resolution stream. I may consider adding this in a future version.

**Reply by kwanm84** [reply]
Thanks for the quick response (and Frigate itself!). 

I have a hardware NVR recording the main stream 24/7. I wanted to use frigate for better object detection, HA integration, and a 2nd backup of cameras. I'll either wait to look at clips on my laptop at high res, or just record in the low res and double check with the main NVR if I need to. So while I don't think I'll run a 3rd NVR, if the feature is added I'll probably use it, but its hardly a deal breaker for me.

**Reply by jasonpstokes** [reply]
What some others have done is set up each physical camera as two cameras in Frigate: one with H264 streams and the other H265. It's not elegent, but it's a clever workaround!

**Comment by kwanm84** [comment]
Thanks

I had considered that, but have 10 cameras. Maybe I could just do it for
the main front camera.

I'll have to see how much nuc performs once I have all 10 set up after the
coral arrives. Just running 2 at the moment.
